cli/command/service/logs.go

```diff
@@ -49,13 +49,14 @@ func newLogsCommand(dockerCli *command.DockerCli) *cobra.Command {
 			opts.target = args[0]
 			return runLogs(dockerCli, &opts)
 		},
-		Tags: map[string]string{"experimental": ""},
```

Comment:
Ok, thinking what we should put here (bear with me, as this is still "new");

When talking to an older daemon, it should only be able to use this command if "experimental" is enabled. I guess that's handled by the daemon just returning a 404 if it's not enabled

However, it "officially" became a feature in docker 17.05 now, so should this be, like this?

    Tags:  map[string]string{"version": "1.29"},


Thinking of new clients talking to older daemons. It's tricky, because it's still possible that we're talking to an older daemon with experimental features enabled 😓 

ping @vieux wdyt?